src/schema/firefox-schemas-import.js

```diff
@@ -0,0 +1,57 @@
+import fs from 'fs';
+
+import commentJson from 'comment-json';
+
+import { processSchemas } from './firefox-schemas';
+
+const SKIP_SCHEMAS = [
+  'native_host_manifest.json',
+];
+
+function readSchema(path, file) {
+  return commentJson.parse(
+    fs.readFileSync(`${path}/${file}`, 'utf-8'),
+    null, // reviver
+    true, // remove_comments
+  );
+}
+
+function writeSchema(path, file, schema) {
+  fs.writeFile(
+    `${path}/${file}`,
+    `${JSON.stringify(schema, undefined, 2)}\n`);
```

Comment:
The newline isn't strictly necessary but we put newlines at the end of our other files and they show up as missing a new line in git if it isn't there.